src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -231,21 +235,18 @@ export const SiteLaunchChecklistBody = ({
       handleDecrementStepNumber()
     }
   }
+  const allTasksDone: boolean =
+    siteLaunchStatusProps?.siteLaunchStatus === "LAUNCHING" ||
```

Comment:
hmm I do [prefer not to use enums](https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh#:~:text=Excessive%20use%20of%20regular%20ENUMs,new%20values%20can%20be%20added.), is there a reason for the use of enum objects?

Reply:
Not enums, but enum objects, 

as in the link:
> Instead of using ENUMs, it's better to opt for objects or types. Objects are flexible and scalable, which means that they can be modified at runtime and new values can be added. 

Objects give us a namespace to work with which looks cleaner, avoids manual typing mistakes, allows VSCode to auto-complete + it is in line with best practices (i.e. if you had to change it to lowercase or smth, you can just do it at 1 place instead of everywhere)

Reply:
ok done e150e0b5c1616041bdc82525f02211bad00b4af6